backend/corpora/dataset_processing/h5ad_data_file.py

```diff
@@ -187,41 +174,30 @@ def validate_anndata(self):
 
     def extract_anndata_elements_from_file(self):
         logging.info(f"Reading in AnnData dataset: {path.basename(self.input_filename)}")
-        self.anndata = anndata.read_h5ad(self.input_filename, backed="r" if self.backed else None)
+        self.anndata = anndata.read_h5ad(self.input_filename)
         logging.info("Completed reading in AnnData dataset!")
 
         self.obs = self.transform_dataframe_index_into_column(self.anndata.obs, "obs", self.obs_index_column_name)
-        self.var = self.transform_dataframe_index_into_column(self.anndata.var, "var", self.vars_index_column_name)
+        self.var = self.transform_dataframe_index_into_column(self.anndata.var, "var", self.var_index_column_name)
 
     def extract_metadata_about_dataset(self):
         """
         Extract metadata information about the dataset that upon conversion will be saved as group metadata with the
         CXG that is generated. This metadata information includes Corpora schema properties, the dataset title and
         a link that details more information about the dataset.
         """
+        self.corpora_properties = self.get_corpora_properties()
 
-        self.corpora_properties = self.get_corpora_properties() if self.use_corpora_schema else None
-
-        if self.corpora_properties is None and self.use_corpora_schema:
+        if self.corpora_properties is None:
             # If the return value is None, this means that we were not able to figure out what version of the Corpora
             # schema the object is using and therefore cannot extract any properties.
             raise ValueError("Unknown source file schema version is unsupported.")
 
         # The title and about properties of the dataset are set by the following order: if they are explicitly defined
-        # then use the explicit value. If the dataset is a Corpora-schema based schema, then extract the title and about
-        # from the corpora_properties. Otherwise, use the input filename (only for title, about will be blank).
-        if self.corpora_properties:
-            corpora_project_links = self.corpora_properties.get("project_links", [])
-            corpora_about_link = next(
-                (link for link in corpora_project_links if (link.get("link_type", None) == "SUMMARY")), {}
-            )
-        else:
-            corpora_about_link = {}
+        # then use the explicit value. Otherwise, use the input filename (only for title, about will be blank).
 
         filename = path.splitext(path.basename(self.input_filename))[0]
-
-        self.dataset_title = self.dataset_title if self.dataset_title else corpora_about_link.get("link_name", filename)
-        self.dataset_about = self.dataset_about if self.dataset_about else corpora_about_link.get("link_url")
+        self.dataset_title = self.dataset_title or filename
```

Comment:
Checked the frontend code -- if there is a title in `corpora_properties` then it prefers that title. Otherwise it uses the top-level title. This is a bit of a legacy design where the CXG conversion was built before we had a "corpora schema" and then once we had a schema, we heavily reduced the number of non-compliant H5ADs to convert.

Reply:
Got it, thx.  For now, I've noted in my arch vision notes that we might consider moving away from having any "Corpora Schema"-awareness in CXG.  It's added a lot of confusion to what a valid CXG file should contain, and where it should be stored. The issue is also now noted in the [CXG spec](https://github.com/chanzuckerberg/single-cell-explorer/blob/main/dev_docs/cxg.md#corpora-schema-encoding).

Reply:
^^ this is definitely the way to go in the future. Thanks for noting it!